app/Database/16-07-27-weekly-summary.txt

```diff
@@ -0,0 +1,22 @@
+16-07-27 Weekly summary (brief).
```

Comment:
sorry. I thought all of this was supposed to be part of the same refactoring, and you wanted it all to be one issue. I guess it wasn't a very smart place for an update .. I will take it out.